recipes/sctools/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   number: 0
-  skip: True # [osx, win]
```

Comment:
We generally don't build on Windows, so we can omit it here.
(NB: the actual syntax would be `[osx or win]`.)